cross_rect_target_pos:
- id: 0
  x: 42.0
  y: 2.5
  z: 1.0

- id: 1
  x: 42.0
  y: 7.5
  z: 1.0

- id: 2
  x: 42.0
  y: 12.5
  z: 1.0

- id: 3
  x: 42.0
  y: 17.5
  z: 1.0

- id: 4
  x: 42.0
  y: 22.5
  z: 1.0

- id: 5
  x: 42.0
  y: 27.5
  z: 1.0

after_rect_target_pos:
- id: 0
  x: 48.0
  y: 2.5
  z: 1.0

- id: 1
  x: 45.0
  y: 7.5
  z: 1.0

- id: 2
  x: 42.0
  y: 12.5
  z: 1.0

- id: 3
  x: 42.0
  y: 17.5
  z: 1.0

- id: 4
  x: 45.0
  y: 22.5
  z: 1.0

- id: 5
  x: 48.0
  y: 27.5
  z: 1.0

leave_room_target_pos:

- x: 48.0
  y: 15.0
  z: 1.0

- x: 63.0
  y: 15.0
  z: 1.0

- x: 63.0
  y: 15.0
  z: 4.0

out_of_room_target_pos:

- id: 1
  x: 78.0
  y: 15.0
  z: 4.0

- id: 2
  x: 75.0
  y: 15.0
  z: 4.0

- id: 3
  x: 72.0
  y: 15.0
  z: 4.0

- id: 4
  x: 69.0
  y: 15.0
  z: 4.0

- id: 5
  x: 66.0
  y: 15.0
  z: 4.0

- id: 6
  x: 63.0
  y: 15.0
  z: 4.0